README.md

```diff
@@ -1,4 +1,7 @@
 # gcc-explorer
 My attempt at making GCC error logs more readable
 
+![image](https://user-images.githubusercontent.com/28424206/159104561-22e774d1-a659-4e7c-9259-7d457e2ab1d5.png)
```

Comment:
Why is there a picture of a face? Is this him?

Reply:
It was an accident new github ui shows weird on my monitor 🙄

Reply:
thats pretty sus ඞඞ

Reply:
> 

what?